CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Added class for Measure report evaluation which will be used in ANC application
+- Added Condition resource to sync params list
```

Comment:
Could you add the _Module name and Pipe_ (`ANC | `) prefix in front of the entry?

i.e. change to `ANC | Added Condition resource to sync params list`